### PR TITLE
docs(hotfix): changelog 1227

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Changelog
+## 3.1.1 - Dec 27, 2021
+This is a hotfix version.
+
+The Azure Bot service is upgraded and does not support some old NodeJs version in some regions any more. This hotfix solve the problem that bot service is not working in those regions which does not support old NodeJs version.
```
Comment (IvanJobs) [inline packages/vscode-extension/CHANGELOG.md:5]: old NodeJs version + s?
This hotfix solve + s?

File: packages/vscode-extension/WHATISNEW.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 - [2.10.0 - Nov 15 2021](#2100---nov-15-2021)
 - [2.9.0 - Nov 01 2021](#290---nov-01-2021)
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+## 3.1.1 - Dec 27, 2021
+This is a hotfix version.
+
+The Azure Bot service is upgraded and does not support some old NodeJs versions in some regions any more. This hotfix solves the problem that bot service is not working in those regions which does not support old NodeJs version.
+
```
Comment (IvanJobs) [inline packages/vscode-extension/WHATISNEW.md:13]: A missed one for `old NodeJs version + s`, :)

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # Changelog
+## 3.1.1 - Dec 27, 2021
+This is a hotfix version.
+
+The Azure Bot service is upgraded and does not support some old NodeJs versions in some regions any more. This hotfix solves the problem that bot service is not working in those regions which does not support old NodeJs version.
```
Comment (eriolchan) [inline packages/vscode-extension/CHANGELOG.md:5]: Actually it is not Azure Bot service, but Azure App Service instead.

Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md]: thank you, fixed.

**Review by eriolchan — APPROVED** [review]

File: packages/vscode-extension/CHANGELOG.md (hunk 3 above)
Comment (sffamily) [reply packages/vscode-extension/CHANGELOG.md]: change all "old" -> "older"

Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md]: ok, fixed.

**Pull request merged**